### DISCUSSION TITLE
Collapsing buttons

### DISCUSSION THREAD

**Discussion by psinger** [h2oai/wave#972] [Q&A]
Is there some way to collapse multiple buttons to a dropdown?
Let's say I have Buttons: "Action A", "Action B", "Action C" and I want to have a dropdown with main title "Action", and sub-options "A", "B", "C".
From my understanding, only `ui.command` can do that now, but cannot place that next to other buttons.

**Comment by mturoci** [comment]
Is `ui.toolbar` what you are looking for?

**Reply by psinger** [reply]
Maybe, is there somewhere an example? But I would need to put buttons next to these collapsed buttons.

**Reply by lo5** [reply]
Something like this? 
https://wave.h2o.ai/docs/examples/toolbar
